apps/app-article/src/article-service.js

```diff
@@ -11,13 +11,13 @@ const articleApi = {
 
     },
     getLatestArticles() {
-        return fetch(config.apiUrl + "latest?start=0&limit=10", {
+        return fetch(config.apiUrl + "latest?start=0&limit=10&paper=" + getBrandValueParam() + "", {
```

Comment:
Why the empty string at the end? 🤔

Reply:
Thats a good question, will fix no need for it 😀